### PR TITLE
Quix-46319: Update quix-samples code segments to 0.5.7

### PR DESCRIPTION
**Pull request by WoozChucky**
Update dotnet project samples to use version 0.5.7 of SDK.
Update dotnet projects to latest net7.0.
Update python projects to use version 0.5.7 of SDK.

### CODE REVIEW

File: csharp/advanced/Bridge.Codemasters/Bridge.Codemasters.Quix/Bridge.Codemasters.Quix.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <ItemGroup>
         <PackageReference Include="Quix.Sdk.Logging" Version="0.4.10-20230127.2"/>
```
Comment (peter-quix) [inline csharp/advanced/Bridge.Codemasters/Bridge.Codemasters.Quix/Bridge.Codemasters.Quix.csproj:8]: Logging

Comment (WoozChucky) [reply csharp/advanced/Bridge.Codemasters/Bridge.Codemasters.Quix/Bridge.Codemasters.Quix.csproj]: Removed all references of Quix.Sdk.Logging package.

**Comment by peter-quix** [comment]
we need to update this to 0.5.6

**Comment by peter-quix** [comment]
Pending testing

**Comment by peter-quix** [comment]
Update: should use 0.5.7, as 0.5.6 has a python flag incorrectly set and by default doesn't start consuming without setting it explicitly to true.

**Review by peter-quix — APPROVED** [review]

**Pull request merged**